sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_items_daily_v1/backfill.yaml

```diff
@@ -4,8 +4,9 @@
   reason: See https://mozilla-hub.atlassian.net/browse/DENG-9289
   watchers:
   - ctroy@mozilla.com
+  - chtroy@mozilla.com
   status: Initiate
   shredder_mitigation: false
-  override_retention_limit: false
+  override_retention_limit: true
```

Comment:
While this will allow the backfill back to 2022-06-21 to proceed, I'm not sure why you'd want to run that when this ETL's source table `moz-fx-data-shared-prod.firefox_desktop_stable.newtab_v1` has 775 day partition retention, so you won't get any data prior to that (i.e. the backfill error about retention is indeed accurate in this case).

IMO the more appropriate thing to do would be adjust the `start_date` to 2023-06-22 (or depending on the timing, maybe 2023-06-23 since we're about to cross the UTC date boundary in a few minutes) instead of enabling `override_retention_limit`.

Reply:
This makees total sense to me. I'll adjust now to do that instead. Thank you!